docs/agents/sematext-agent/containers/installation.md

```diff
@@ -0,0 +1,102 @@
+To deploy Sematext Agent container run the following command on each host:
+
+```bash
+docker run -d  --restart always --privileged -P --name st-agent \
+-v /sys/kernel/debug:/sys/kernel/debug \
+-v /var/run/:/var/run/ \
+-v /proc:/host/proc:ro \
+-v /etc:/host/etc:ro \
+-v /sys:/host/sys:ro \
+-v /usr/lib:/host/usr/lib:ro \
+-e CONTAINER_TOKEN==<Docker App Token> \
+-e INFRA_TOKEN=<Infra App Token> \
+-e REGION=US \
+-e JOURNAL_DIR=/var/run/st-agent \
+-e LOGGING_WRITE_EVENTS=false \
+-e LOGGING_REQUEST_TRACKING=false \
+-e LOGGING_LEVEL=info \
+-e NODE_NAME=`hostname` \
+-e CONTAINER_SKIP_BY_IMAGE=sematext \
+sematext/agent:latest
+```
+
+Besides providing several bind mounts for Docker socket, _procfs_ and journal directory, tokens are required to ship data to the right place. Sematext Agent will gather data about running processes on the system, basic operating system metrics, machine/instance related information, and ship it to the Infra App token. It will also detect active containers and start collecting different container metrics such as memory usage, network I/O statistics, disk throughput, etc.
+
+By default, the US region receiver endpoints are used to ship data to Sematext Cloud. You can override receiver addresses by either passing `SERVER_BASE_URL` for metrics receivers, `LOGS_RECEIVER_URL` and `EVENTS_RECEIVER_URL` for log and event receivers respectively or specify an alternative region via `REGION` environment variable.
```

Comment:
Btw. what happens if one uses both, and uses conflicting values, like REGION=US, but pointing to EU logs receiver?  If one specifies the region, do we override any manually set receiver URLs?  If yes, should we document this?  If not, should we?

Reply:
Maybe also mention which alternative `REGION` value is possible (just EU for now).

Reply:
>  If yes, should we document this? If not, should we?

If any of the `*URL` are provided, then `REGION` has no effect. I'll explicitly point that out.